tests/data/out_teacup.csv

```diff
@@ -0,0 +1,242 @@
+Time,Characteristic Time,Heat Loss to Room,Room Temperature,Teacup Temperature
```

Comment:
I guess I saw this in the other or (to this branch) - what does this file do? should it be ignored?

Reply:
This file and other 2 are used to test reading outputs from csv in "regular" and "transposed" way and comparing them, one of them has different values to test the raising errors and raising warnings with assert_frames_close.
They are used in unit_test_benchmarking.py